.github/workflows/pr-title-check.yml

```diff
@@ -0,0 +1,47 @@
+name: "Lint PR"
+
+on:
+  pull_request_target:
```

Comment:
If I have to be honest. I feel a bit uneasy on using `pull_request_target` due to the possible security implications. This needs to be handled with careful considerations in the future.  We should add comments in line 1 about future changes to this file, carefully review updates to  `amannn/action-semantic-pull-request@01d5fd8a8ebb9aafe902c40c53f0f4744f7381eb` .

No checkout actions, scripts, or builds should be added to this workflow.  Permissions should always be used with extreme caution.

https://securitylab.github.com/research/github-actions-preventing-pwn-requests/
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target

This is not a blocker, it just made my tin foil hat activate.